src/main/webapp/js/omniComment.js

```diff
@@ -217,18 +218,16 @@ $(document).ready(function() {
     }
     
     function showCommentAndItsPanel(comment) {
-        $(comment).show();
+        var commentToShow = $(comment);
+        commentToShow.show();
         
         // to show feedback question + feedback session panel
         // if not all list elements are hidden within fbResponse, then show fbResponse
-        if ($(comment).prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
-            if ($(comment).parent().find('li[style*="display: none"]').length !== $(comment).parent().find('li').length) {
-                var commentListRegionForFeedbackResponse = $(comment).closest('tr');
-                // a fbResponse in instructorCommentsPage (html) is made up of 4 rows as the followings
-                commentListRegionForFeedbackResponse.show();
-                commentListRegionForFeedbackResponse.prev().show();
-                commentListRegionForFeedbackResponse.prev().prev().show();
-                commentListRegionForFeedbackResponse.prev().prev().prev().show();
+        if (commentToShow.prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
+            if (commentToShow.parent().find('li[style*="display: none"]').length !== commentToShow.parent().find('li').length) {
```

Comment:
Can use explaining variables to clarify the meaning of these boolean conditions?
Also, looks like you can `and`  the two conditions (because there are no `else`) to reduce nesting.